tests/curators/test_curators_multivalue.py

```diff
@@ -0,0 +1,48 @@
+import bionty as bt
+import lamindb as ln
+import pandas as pd
+import pytest
+from lamindb.core.exceptions import ValidationError
+
+
+@pytest.fixture
+def df():
+    return pd.DataFrame(
+        {
+            "sample_id": [["sample1", "sample2"], ["sample2"], ["sample3"]],
+            "dose": [[1.2, 2.3], [1.2], [2.3]],
+            "cell_type": [["B cell", "T cell"], ["B cell"], ["T cell"]],
+            "tissue": [["blood", "pulmo"], ["blood"], ["lung"]],
+        }
+    )
+
+
+@pytest.fixture(scope="module")
+def lists_schema():
+    schema = ln.Schema(
+        name="lists schema cat",
+        features=[
+            ln.Feature(name="sample_id", dtype=str).save(),
```

Comment:
This should be typed as `list[str]` not as `str`.

Reply:
No, it's always typed with the individual value.